tools/wpt/browser.py

```diff
@@ -666,19 +666,16 @@ def install_webdriver(self, dest=None, channel=None, browser_binary=None):
         return find_executable("msedgedriver", dest)
 
     def version(self, binary=None, webdriver_binary=None):
-        if uname[0] != "Windows":
-            try:
-                version_string = call(binary, "--version").strip()
-            except subprocess.CalledProcessError:
-                self.logger.warning("Failed to call %s" % binary)
-                return None
-            m = re.match(r"(?:MSEdge|Edge) (.*)", version_string)
-            if not m:
-                self.logger.warning("Failed to extract version from: %s" % version_string)
-                return None
-            return m.group(1)
-        self.logger.warning("Unable to extract version from binary on Windows.")
-        return None
+        try:
+            version_string = call(binary, "--version").strip()
```

Comment:
On Windows this will launch Edge process but doesn't return a version info. We could use PowerShell on Windows to get version information.

Reply:
Oh... this was written to work on macOS, not Windows. I should probably close this then, but the problem to fix is probably here as @Hexcles suggested.

Reply:
We can probably update this PR to use PowerShell on Windows, something like this should work:

        command = "(Get-Item ‘%s’).VersionInfo.FileVersion" % binary
        try:
            return call("powershell.exe", command).strip()
        except (subprocess.CalledProcessError, OSError):
            self.logger.warning("Failed to call %s in PowerShell" % command)
            return None